src/api-reference/direct-connects/hotel-service-2/Availability.markdown

```diff
@@ -187,7 +199,7 @@ The maximum allowed size of `OTA_HotelAvailRS` is 5 MB. Any response that exceed
             </RoomType>
           </RoomTypes>
           <RatePlans>
-            <RatePlan RatePlanID="XNFYP4I" AvailabilityStatus="ChangeDuringStay">
+            <RatePlan RatePlanID="XNFYP4I" AvailabilityStatus="ChangeDuringStay" RatePlanType="4">
```

Comment:
CWT is already using RatePlanType now, but not compliant to OTA standards

RatePlanType="Refundable" which is not correct.